server/app/forms/MapQuestionForm.java

```diff
@@ -126,14 +141,26 @@ private void setFormWithQuestionSettings(ImmutableSet<QuestionSetting> settings)
         getSettingFromQuestionSettings(settings, MapSettingType.LOCATION_DETAILS_URL_GEO_JSON_KEY);
 
     this.filters = new ArrayList<>(getFiltersFromQuestionSettings(settings));
+    this.locationTag =
+        getSettingFromQuestionSettings(settings, MapSettingType.LOCATION_TAG_GEO_JSON_KEY);
   }
 
   /** Converts {@link QuestionSetting} back to form {@link Setting} for editing. */
   private Setting getSettingFromQuestionSettings(
       ImmutableSet<QuestionSetting> settings, MapSettingType type) {
     return settings.stream()
         .filter(setting -> setting.settingType().equals(type))
-        .map(setting -> new Setting(setting.settingValue(), ""))
+        .map(
```

Comment:
(question) Will/should you only ever have a single setting for a specific type? If there are more than one you are losing it with the findFirst.